src/Gui/Stories/Collapsible.story.luau

```diff
@@ -0,0 +1,98 @@
+local Hoarcekat = script:FindFirstAncestor("Hoarcekat")
+
+local React = require(Hoarcekat.Packages.React)
+local ReactRoblox = require(Hoarcekat.Packages.ReactRoblox)
+
+local Assets = require(Hoarcekat.Plugin.Assets)
+
+local Collapsible = require(Hoarcekat.Plugin.Gui.Components.Collapsible)
+local IconListItem = require(Hoarcekat.Plugin.Gui.Components.IconListItem)
+
+local e = React.createElement
+
+local function CollapsibleChildrenFragment(props: {
```

Comment:
This is simple enough to just inline in `TestCollapsibleStory`, but it's fine if you don't